src/plugins/validate-semantic/validators/oas3/refs.js

```diff
@@ -29,3 +29,31 @@ export const validateOAS3RefsForRequestBodiesReferenceRequestBodyPositions = ()
       }, [])
     })
 }
+
+export const validateOAS3RefsForHeadersReferenceHeadersPositions = () => sys => {
+  return sys.validateSelectors
+    .allParameters()
+    .then(nodes => {
+      return nodes.reduce((acc, node) => {
+        const value = node.node
+        const ref = value.$ref
+
+        if(!ref) {
+          return acc
+        }
+        
+        // Ignore external references
+        if (ref.startsWith("#/")) {
```

Comment:
This check is not needed because it's covered by the next `if`.

Reply:
Done!